### DISCUSSION TITLE
Jupyter “Code” preview breaks in New Code Search and Code View (Beta)

### DISCUSSION THREAD

**Discussion by pan93412** [community/community#40152] [Projects and Issues]
### Select Topic Area

Bug

### Body

The Jupyter Notebook preview throws “Error – Looks like something went wrong!” after clicking “Code” button. Refresh the page workarounds this issue. It happens on Safari & Edge.

## Example Repository

- Folder: https://github.com/Yorko/mlcourse.ai/tree/main/jupyter_chinese
- File: https://github.com/Yorko/mlcourse.ai/blob/main/jupyter_chinese/topic01-使用-Pandas-进行数据探索.ipynb

## Screencast

https://user-images.githubusercontent.com/28441561/204091106-ed5e13de-ef4c-4e3d-aca5-e60828640420.mov

**Comment by ossanna16** [comment]
Hi there @pan93412 and welcome to our community! Thank you for reporting this bug!

**Comment by pan93412** [comment]
Seems like it has been fixed!
